tests/plugins/crop_generators/test_autoencoder_crop_generator.py

```diff
@@ -25,56 +26,34 @@ def out_dir(tmpdir):
 
 @pytest.fixture(scope='function')
 def config(out_dir):
-    return {
-        "image_set": {
-            "mask_objects": False,
-            "channels": ['R', 'G', 'B'],
-            "width": 128,
-            "height": 128,
-            "path": out_dir
-        },
-        "sampling": {
-            "images": 12,
-            "box_size": 16,
-            "locations": 10,
-            "locations_field": 'R'
-        },
-        "training": {
-            "minibatch": 2,
-            "output": out_dir
-        },
-        "validation": {
-            "minibatch": 2,
-            "output": out_dir
-        },
-        "queueing": {
-            "loading_workers": 2,
-            "queue_size": 6
-        },
-        "model": {
-            "type": "mixup",
-            "alpha": 0.2,
-            "sequence_length": 5,
-            "params": {
-                "batch_size": 2
-            }
-        }
-    }
+    with open("tests/files/config/test.json", 'r') as f:
+        config = json.load(f)
+    for path in config["paths"]:
+        config["paths"][path] = out_dir + config["paths"].get(path)
+    config["paths"]["root_dir"] = out_dir
+    return config
+
+@pytest.fixture(scope='function')
+def make_struct(config):
+    for key, path in config["paths"].items():
+        if key not in ["index", "config_file", "root_dir"]:
+            os.makedirs(path+"/")
+    return
 
 
 @pytest.fixture(scope='function')
-def metadata(out_dir):
-    filename = os.path.join(out_dir, 'metadata.csv')
+def metadata(out_dir, make_struct):
+    filename = os.path.join(out_dir, 'index.csv')
     df = pd.DataFrame({
         'Metadata_Plate': __rand_array(),
         'Metadata_Well': __rand_array(),
         'Metadata_Site': __rand_array(),
         'R': [str(x) + '.png' for x in __rand_array()],
         'G': [str(x) + '.png' for x in __rand_array()],
         'B': [str(x) + '.png' for x in __rand_array()],
+        'Class': ['0', '1', '2', '3', '0', '1', '2', '3', '0', '1', '2', '3'],
         'Sampling': [0, 1, 0, 1, 0, 1, 0, 1, 0, 1, 0, 1],
-        'Split': [0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1],
-        'Target': [None, None, None, None, None, None, None, None, None, None, None, None]  # TODO: rewrite autoencoder crop generator so that this is not necessary
+        'Split': [0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1]
```

Comment:
does this still work without setting targets? I have't updated autoencoder crop generator yet